### PR TITLE
Remove decode() as its not needed for versions 3.8 +

### PR DESCRIPTION
**Pull request by m7salam**
this should prevent the error of the str in decode we get if using version 3.8.5 or above

### CODE REVIEW

**Comment by m7salam** [comment]
its related to this issue https://github.com/CuriousLearner/django-phone-verify/issues/51

**Comment by CuriousLearner** [comment]
Hi @m7salam 👋 

Thank you for the PR.

The CI is failing https://travis-ci.com/github/CuriousLearner/django-phone-verify/builds/215598946#L351

Let me know once you fix it and I'll take a look at this and test the fix for various python versions.

Many thanks!

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "python.formatting.provider": "black"
+}
```
Comment (CuriousLearner) [inline .vscode/settings.json:3]: While, you're at it, please remove this config file.

**Comment by m7salam** [comment]
@CuriousLearner its already ready and passed the checks :)

File: phone_verify/backends/base.py [hunk 2]
```diff
@@ -49,7 +49,9 @@ def generate_session_token(cls, phone_number):
         identifying a particular device in subsequent calls.
         """
         data = {"phone_number": phone_number, "nonce": random.random()}
-        return jwt.encode(data, django_settings.SECRET_KEY).decode()
+        # if (sys.version_info.major == 3 and sys.version_info.minor >= 7):
+        #     return jwt.encode(data, django_settings.SECRET_KEY)
```
Comment (CuriousLearner) [inline phone_verify/backends/base.py:53]: This fix is commented out.

**Review by CuriousLearner — CHANGES_REQUESTED** [review]

**Comment by CuriousLearner** [comment]
This had the fix commented out and is now superseded by #58 

Thanks for your efforts @m7salam !

**Comment by m7salam** [comment]
@CuriousLearner the comment wasn't the solution it was the old one and I commented it when I solved it by following by updating a certain library you were using I found the updated version solved the problem so I made the update and pushed it and it succeeded all the tests 

So the comments are useless it just needs to be removed and I'm sorry I didn't have time between work and got sick to remove it and push it 

It's already passing all your tests and working well

**Comment by CuriousLearner** [comment]
Ok, no problem. I think everything will work from #58. I'm just waiting for CI to pass for each of the commits. Currently it seems like travis is having some issues and cancelling the builds mid-way.

So, I'm gonna fix those and release a new version.

Anyways, thanks for your time. I really appreciate it. :)

**Pull request closed without merging**